src/components/ButtonsContainer/index.jsx

```diff
@@ -28,6 +28,7 @@ const useStyles = makeStyles((theme) => ({
         display: 'inline-block',
         textAlign: 'center',
         background: 'none',
+        color: '#fff',
```

Comment:
Could you change this to `theme.palette.text.primary` like in:

https://github.com/roerohan/8086.js/blob/7f399c38e3c61de35a48450db17920b5e51857b6/src/components/Emulator/index.jsx#L11

Reply:
Also, would be nice if you could PR to the dev branch instead of master.

Reply:
mad all the necessary changes based on the instructions 👍🏻